wiki/Plotting-Velocity-Models.md

```diff
@@ -75,3 +75,13 @@ plot-velocity-model vm-params vm_params.yaml --title "Swedge1: Density Plot"  --
 ```
 
 ![](images/swedge1.png)
+
+# I Want a 3d View of the Velocity Model
+
+The `view-velocity-model` tool can read a velocity model file and view its contents interactively in 3D. This may not run well if your model is particularly large or if you have a slow computer -- especially if you move the slices around. You can also configure the available slice controls between xy, yz, and xz slices. You must know the number of slices in the model's x, y and z axes. These you can either read from the `vm_params.yaml` for the old workflow, or from the realisation json file in the new workflow.
+
+``` bash
+view-velocity-model Velocity_Model/rho3dfile.(d|p|s) NX NY NZ
```

Comment:
```suggestion
view-velocity-model Velocity_Model(rho|vp|vs)3dfile.(d|p|s) NX NY NZ
```

Reply:
tbh that documentation is a little lazy of me anyway since a researcher isn't going to be able to read that regex easily. I'll update it properly.